### DISCUSSION TITLE
"Document update conflict" when trying to update User

### DISCUSSION THREAD

**Discussion by m7kael** [Budibase/budibase#8081] [Help] [answered]
Hi,
After udpdate to 2.0 I am getting "Document update conflict" when trying to update a customer row in an app. I normally create and update users with public api but I also get the error when trying to manually update a user. Any tips or tricks? 
<img width="400" alt="Screenshot 2022-10-01 at 09 49 41" src="https://user-images.githubusercontent.com/112936118/193397278-95329a0f-ddd0-4f9b-af34-a353908bf250.png">

**Comment by dominik-bsr** [comment]
Same here ..
Added a custom column to the User table, getting a "Document Update Conflict" when I'm trying to add data

**Comment by validlogin** [comment]
Also experiencing this issue, unfortunately.

**Comment by feurer98** [comment]
same error for me

**Comment by melohagan** [comment, marked as answer]
Sorry guys! Bug raised here: https://github.com/Budibase/budibase/issues/8043

**Reply by validlogin** [reply]
Any updates on this or ETA for a fix?

Thanks!
